### ISSUE TITLE
where is sigmoid?

### ISSUE THREAD

**Issue by kwea123** [yhw-yhw/PVAMVSNet#3]
you use `volumegatelight` here
https://github.com/yhw-yhw/PVAMVSNet/blob/05084abac2512a367a81558785fb8d7e83b4ed0f/models/vamvsnet.py#L455-L459

which is defined as
https://github.com/yhw-yhw/PVAMVSNet/blob/05084abac2512a367a81558785fb8d7e83b4ed0f/models/submodule.py#L75-L80

but there is no `sigmoid`. Generally the attention is something between 0 and 1, and in your paper you also have sigmoid function in the supplementary material table 2. However I cannot find any sigmoid in the code; if the attention is without sigmoid in my opinion it is strange. Does it also work without sigmoid? Or did I miss anything?

**Comment by yhw-yhw** [comment]
Yes, it did work, we also did the experiment with using sigmoid, but replacing it with ReLU as in our code and our provided model, the performance is slightly better than the model with using sigmoid. We guess that in our Voxel-wise view aggregation, we consider that only using weight map to enhance the  feature map is better than highlight and suppress. By the way, we will provide the 2D Pixel-wise view aggregation in the future and update our code base with new results.

**Comment by kwea123** [comment]
Yes, the current code misses pixelVA; but it seems that it doesn't contribute much? There is `+ VoxelVA + CTF + MMP` in Table 2, but no `+ VoxelVA + PixelVA + CTF + MMP`.

Also in Figure 9., you have the same color for `Baseline` and `+VoxelVA`, if you can fix that it'll be good.

**Comment by yhw-yhw** [comment]
Our VoxelVA and  PixelVA are two different self-adaptive view aggregation for the same target, in our experiments, VoxelVA is better than PixelVA in 3DCNN but PixelVA still have improvements on the performance, it is useful for the situation that you do not have 3D feature volume like in R-MVSNet.
